public/scripts/main.js

```diff
@@ -273,38 +383,109 @@ container.addEventListener('keyup', function(event) {
   keys[event.keyCode] = false;
 });
 
+// Skills
+/*
+player.skillA1 = new Skills({
+  description: 'aa',
+  damage: -5,
+});
+*/
+player.skillA1 = skilldefault.Skill1;
+
+function fightLoop(subject, opponent) {
```

Comment:
a fighting sequence order base on character.fightspeed, this only implement for one turn (player -> npc or opposite) since it is triggered by key press event.